src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -117,14 +137,22 @@ export enum MutationCountBy {
 export type AxisMenuSelection = {
     entrezGeneId?:number;
     genesetId?:string;
+    treatmentId?:string;
     selectedGeneOption?:{value:number, label:string}; // value is entrez id, label is hugo symbol
+    selectedDataSourceOption?:{value:string, label:string};
     selectedGenesetOption?:{value:string, label:string};
+    selectedTreatmentOption?:{value:string, label:string};
     dataType?:string;
     dataSourceId?:string;
     mutationCountBy:MutationCountBy;
     logScale: boolean;
 };
 
+export type UtilitiesMenuSelection = {
+    entrezGeneId?:number;
```

Comment:
Can you adjust the name to describe what the purpose is of selecting a gene in the utilities menu? e.g. highlightedEntrezGeneId or something like that?

I don't also see the purpose of storing both selectedGeneOption and entrezGeneId, seems like you could compute entrezGeneId from the selectedGeneOption, and indeed you could do that (see how getters are handled in AxisMenuSelection)

Reply:
How about `geneForMutationAndCnaStyling`?

Reply:
thats great :+1: